schedule/yast/opensuse/yast2_gui/yast2_gui.yaml

```diff
@@ -15,20 +15,27 @@ schedule:
   - console/prepare_test_data
   - console/consoletest_setup
   - console/hostname
+  - console/install_packages_simple
```

Comment:
why we need this two modules here if they are done in the parent:
https://openqa.opensuse.org/tests/2843930#step/install_packages_simple/1
same for sle:
https://openqa.suse.de/tests/9825529#step/install_packages_simple/1